packages/design-system/src/styles/components/_UsaBanner.scss

```diff
@@ -55,12 +49,12 @@ $usa-banner-font-family: $font-sans;
   align-items: center;
   background-color: $usa-banner-close__background-color;
   display: flex;
-  height: $usa-banner-mobile-close-size;
+  height: 48px;
   justify-content: center;
   margin-left: auto;
   margin-right: -$spacer-2;
```

Comment:
the negatives used in our sass files will need to be evaluated as math i think. when we replace these sass vars with css vars, this won't compile correctly without a `calc`